playground/SplitscreenManager.js

```diff
@@ -0,0 +1,89 @@
+export class SplitscreenManager {
+
+    constructor( editor ) {
+
+        this.editor = editor;
+        this.renderer = editor.renderer;
+        this.composer = editor.composer;
+
+        this.gutter = null;
+        this.gutterMoving = false;
+        this.gutterOffset = 0.75;
+
+    }
+
+    setSplitview( value ) {
+
+		let nodeDOM = this.editor.domElement;
+		let rendererDOM = this.renderer.domElement;
+
+		if ( value ) {
+
+			this.addGutter( rendererDOM, nodeDOM );
+
+		} else {
+
+			this.removeGutter( rendererDOM, nodeDOM );
+
+		}
+
+    }
+
+    addGutter( rendererDOM, nodeDOM ) {
+
+        rendererDOM.style[ "z-index" ] = 20;
+
+        this.gutter = document.createElement( "div" );
+        this.gutter.className = "gutter";
+
+        document.getElementById( "splitview" ).appendChild( this.gutter );
```

Comment:
Would it be possible to use variables instead of obtaining elements by ID? This way you could create the div using javascript instead of adding it to the html.

Reply:
Done :)